src/client/src/components/JobApplicationPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import axios from "axios";
+import ".css/jobApplication.css"
+
+
+const JobApplicationPage = () =>{
+
+    const getJobList = async (e) => {
+        e.preventDefault();
+        axios.get("http://localhost:9000/jobs/:job_id", () =>{
+
+        })
+        .then((response) =>{
+
+        })
+        .catch((error) => {
+
+        })
+
+        }
+
+    }
```

Comment:
Maybe add a comment in what this function does. Also if you can console.log(error) in the catch clause to check for any errors, if any